spring-xsuaa/src/test/java/com/sap/cloud/security/xsuaa/token/XsuaaTokenTest.java

```diff
@@ -307,12 +307,12 @@ private XsuaaToken createToken(JWTClaimsSet.Builder claimsBuilder) {
 		return new XsuaaToken(jwt);
 	}
 
-	private static class SamlExternalAttrClaim {
+	public static class SamlExternalAttrClaim {
```

Comment:
is there a specific reason why this was made public?

Reply:
Yep... this was required by Nimbus, it raised an IllegalArgumentException (RuntimeException) as it could not resolve public members from private class via reflection.